src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
 public class Duke {
+
+    private static Scanner in = new Scanner(System.in);
+    private static final Task[] tasks = new Task[100];
```

Comment:
For program scalability, you might want to consider using `ArrayList` instead of setting a maximum number of tasks user can have here.